docs/source/integrations/building-integrations.md

```diff
@@ -125,36 +96,27 @@ const context: ContextFunction<[ExpressContextFunctionArgument], TContext> =
   options?.context ?? defaultContext;
 ```
 
-The `context` function that we have here will be _called_ in the execution step.
+Note, the `context` function is _called_ during the [execution step](#execute-the-graphql-request).
 
 ### Handle Requests
 
-Integration packages should be implemented as some form of request handler or
-framework plugin. Handlers typically receive information about each request
-including common HTTP parts (i.e. `method, `headers`, and `body`) and likely
-some other useful contextual information. Integrations are responsible for
-mapping this information into a consistent format for Apollo Server.
+We recommend implementing your integration package as either a request handler or a framework plugin. Request handlers typically receive information about each request, including standard HTTP parts (i.e., `method`, `headers`, and `body`) and other useful contextual information.
 
-The request handler has 4 main functions, which we will discuss in detail below:
-1. Parse the request
-2. Construct an `HTTPGraphQLRequest` object from the incoming request
-3. Execute the GraphQL request via Apollo Server
-4. Return a well-formed response to the client
+A request handler has 4 main responsibilities:
+1. [Parse the request](#parse-the-request)
+2. [Construct an `HTTPGraphQLRequest` object](#construct-the-httpgraphqlrequest-object) from the incoming request
+3. [Execute the GraphQL request](#execute-the-graphql-request) using Apollo Server
+4. Return a well-formed [response to the client](#send-the-response)
 
 #### Parse the request
 
-Apollo Server responds to a variety of requests, handling queries via `GET` and
-`POST`, full GraphQL queries and APQs, and landing pages like Explorer.
-Fortunately, this is all part of Apollo Server's core logic and isn't something
-that integrations need to worry about.
+Apollo Server responds to a variety of requests, such as HTTP requests (i.e., `GET` and `POST`), APQs, and landing page requests (e.g., Apollo Sandbox). Fortunately, this is all part of Apollo Server's core logic, and it isn't something integration authors need to worry about.
```

Comment:
All of these are HTTP requests via GET and POST, so to separate them as their own thing from the rest reads a little funny. I realize that's kind of what I already had here as well.
```suggestion
Apollo Server responds to a variety of requests via both `GET` and `POST` such as standard GraphQL queries, APQs, and landing page requests (e.g., Apollo Sandbox). Fortunately, this is all part of Apollo Server's core logic, and it isn't something integration authors need to worry about.
```

Reply:
Thank you for the clarification, I like this rephrase!